### PR TITLE
Tier1 dodrio extension

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: resource_files/UGent/resource_specs.json [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    "nodes": 96,
+    "ppn": 36,
+    "start_date": "2018-01-24"
+  },
+  {
+    "resource": "delcatty",
+    "processors": 2560,
+    "nodes": 160,
+    "ppn": 16,
+    "start_date": "2013-05015"
```
Comment (wpoely86) [inline resource_files/UGent/resource_specs.json:21]:
```suggestion
    "start_date": "2013-05-15"
```

File: resource_files/UGent/resource_specs.json [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    "nodes": 16,
+    "ppn": 36,
+    "start_date": "2020-02-13"
+  },
+  {
+    "resource": "gengar",
+    "processors": 1552,
+    "nodes": 194,
+    "ppn": 8,
+    "start_date": "2008-11-01"
```
Comment (wpoely86) [inline resource_files/UGent/resource_specs.json:63]: `end_date`?

**Review by wpoely86 — APPROVED** [review]

File: resource_files/UGent/resource_specs.json [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+    "nodes": 42,
+    "ppn": 128,
+    "start_date": "2020-10-27"
+  },
+  {
+    "resource": "dodrio_gpu_rome_a100",
+    "processors": 960,
+    "nodes": 40,
+    "ppn": 48,
+    "start_date": "2020-10-27"
```
Comment (WouterVanAssche) [inline resource_files/UGent/resource_specs.json:168]: ppn * nodes = processors
40 * 48 = 1920

**Review by WouterVanAssche — CHANGES_REQUESTED** [review]
If end dates are  known, also add end dates where possible.

**Pull request merged**